docs/adr/0003-confirmation-next-steps.md

```diff
@@ -0,0 +1,52 @@
+# Save and return
+
+- Status: proposed
+- Deciders: FCDO / OS maintainers: [@jenbutongit](https://github.com/jenbutongit) [@superafroman](https://github.com/superafroman) [@alisalaman-cyb](https://github.com/alisalaman-cyb)
+- Date: 2023-04-17
+- Amended: 2023-04-17
+
+## Context and Problem Statement
+
+In some cases there is the necessity to have content flow over multiple lines to help with readability. Currently, you can add custom next steps text by appending the `specialPages.confirmationPage.customText.nextSteps` property to the form configuration object, however there is no way of doing this through the designer UI, and on top of this json strips out new line characters by default.
+
+I think this gives us a good chance to decide how we want to make the next steps text editable, and to also re-evaluate how we print this variable in the nunjucks template.
+
+This will also mark off [#723](https://github.com/XGovFormBuilder/digital-form-builder/issues/723) while we're looking into this.
+
+Related issues:
+
+- [#723](https://github.com/XGovFormBuilder/digital-form-builder/issues/723)
+
+## Considered Options
+
+Option 1:
+
+1. Create a special pages edit component, which we can use in the future for other special page variables, but for the time being will just be used for confirmation page variables.
+2. Next steps field added to component as long text field, allowing users to input text over multiple lines, but not add custom html
+3. Add logic to split the inputted content into multiple strings in an array, using the line breaks as the delimiting character.
+4. Update the form schema to allow an array of strings for the nextSteps variable as well as the current string value.
+5. In the confirmation view template, change the line which populates the "next steps" area to show a partial for the next steps variable. This partial should have the conditional logic to show each line with break tags in between, or the single line string for backwards compatability.
+
+Option 2:
+
+1. Create a special pages edit component, which we can use in the future for other special page variables, but for the time being will just be used for confirmation page variables.
+2. Next steps field added to component as html field based on current html field component.
```

Comment:
I think it would be ideal that we allow this and the summary page to be edited like the question pages, but for speed let's just stick to making it clear that this can be edited as string or html.